src/parsing.jl

```diff
@@ -113,25 +112,24 @@ Base.lastindex(pqv::PQValue) = lastindex(string_view(pqv))
     parse(::Type{T}, pqv::PQValue) -> T
 
 Parse a value of type `T` from a `PQValue`.
-By default, this uses any existing `parse` method for parsing a value of type `T` from a
+By default, this uses any existing `pqparse` method for parsing a value of type `T` from a
```

Comment:
It still does `parse` by default, under the hood, so I'd leave that as is but add a comment saying you can implement default PostgreSQL-specific parsing for a given type by overriding `pqparse`.